content/handbook/product/ux/experience-research/writing-usability-testing-script.md

```
@@ -0,0 +1,158 @@
+- **協働を支援します。** 特に、他の人があなたの作業をより容易にレビューできるようにします。
+- **セッション中の時間管理を助けます。** 約束した長さにリサーチセッションを収めることが重要です。
+
+テストスクリプトなしで堅実なユーザビリティ調査を実践することは、実質的に不可能です。スクリプトなしで臨んではいけません。
+
+## タスクベースのユーザビリティテストの最初のドラフトを書く方法
+
+ユーザビリティ調査の最初のドラフトを書くためのスタート地点として、この [ユーザビリティテストスクリプトテンプレート](https://docs.google.com/document/d/135OtPVzguF8ZmbtfVL9hqVeehcN48IxWceWr5g070T8/copy)（内部アクセスのみ）を使ってください。
+
+ユーザビリティ調査スクリプトは通常 4 つの主要パートに従います。
+
+1. [導入](#introduction)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)